### PR TITLE
api-registration

### PR DESCRIPTION
**Pull request by easton-morris**
image

### CODE REVIEW

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    INSERT INTO "users" ("username", "hashedPassword")
+    VALUES ($1, $2)
+    RETURNING *
+  `;
+      const params = [username, hashedPw];
+      db.query(sql, params)
+        .then(result => {
+          const [user] = result.rows;
+          if (user) {
+            res.status(201).json(user);
```
Comment (scbowler) [inline api-registration/server/index.js]: You don't want to send the password back to the client, even if it is hashed. The best way to control the data being sent back to the client is to specify the desired columns in the `returning` clause of the SQL statement.
```SQL
returning "userId", "username", "createdAt"
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed. Let me know if you have any questions.

File: api-registration/server/index.js (hunk 1 above)
Comment (easton-morris) [reply api-registration/server/index.js]: Oh yeah... woops. 
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**